### PR TITLE
Feat/get region dxf api

### PR DESCRIPTION
**Pull request by matthewAnsys**
Add api for accessing dxf regions from Motor-CAD

### CODE REVIEW

File: src/ansys/motorcad/core/methods/adaptive_geometry.py [hunk 1]
```diff
@@ -64,6 +64,30 @@ def get_region(self, name):
 
         return region
 
+    def get_region_dxf(self, name):
+        """Get Motor-CAD dxf geometry region.
+
+        Parameters
+        ----------
+        name : string
+            name of region.
```
Comment (PipKat) [inline src/ansys/motorcad/core/methods/adaptive_geometry.py:73]:
```suggestion
            Name of the region.
```

File: src/ansys/motorcad/core/methods/adaptive_geometry.py [hunk 2]
```diff
@@ -64,6 +64,30 @@ def get_region(self, name):
...
+
+        Returns
+        -------
+        ansys.motorcad.core.geometry.Region or ansys.motorcad.core.geometry.RegionMagnet
+            Motor-CAD region object.
+
+        """
+        self.connection.ensure_version_at_least("2024.0")
```
Comment (jgsdavies) [inline src/ansys/motorcad/core/methods/adaptive_geometry.py:81]: 2024

Comment (jgsdavies) [inline src/ansys/motorcad/core/methods/adaptive_geometry.py:81]:
```suggestion
        self.connection.ensure_version_at_least("2024.2.0")
```

File: tests/test_geometry.py [hunk 3]
```diff
@@ -178,6 +178,38 @@ def test_get_region():
     assert ("region" in str(e_info.value)) and ("name" in str(e_info.value))
 
 
+def test_get_region_dxf():
+    mc.load_dxf_file(get_dir_path() + r"\test_files\dxf_import.dxf")
```
Comment (jgsdavies) [inline tests/test_geometry.py:182]: I think this file needs to be added to PR

**Review by jgsdavies — CHANGES_REQUESTED** [review]

File: src/ansys/motorcad/core/methods/adaptive_geometry.py (hunk 2 above)
Comment (jgsdavies) [inline src/ansys/motorcad/core/methods/adaptive_geometry.py]:
```suggestion
        self.connection.ensure_version_at_least("2024.2")
```

**Review by jgsdavies — APPROVED** [review]

**Pull request merged**